add READMEs to dapr/proto folders for ease of information (#7095)
dapr/dapr: dapr/proto/sentry/v1/README.md

# Sentry Service APIs

> This folder is intended for Sentry Service APIs that the Daprd Sidecars use to manage mTLS between services and acts as a certificate authority. [See more on the Sentry Control Plane Service in the Dapr Docs](https://docs.dapr.io/concepts/dapr-services/sentry/).

Reviewer: Sentry is used by the Dapr control plane services too

Reviewer 2: ```suggestion
This folder is intended for `sentry` service APIs that the `daprd` sidecars use to manage mTLS between services and acts as a certificate authority. More details about the `sentry` please refer to the [docs](https://docs.dapr.io/concepts/dapr-services/sentry/).
```
